src/NuGet.Core/NuGet.CommandLine.XPlat/Program.cs

```diff
@@ -52,7 +52,11 @@ public static int MainInternal(string[] args, CommandOutputLogger log)
             }
 
             var app = new CommandLineApplication();
+#if IS_CORECLR
```

Comment:
Why this condition? Running `NuGet.CommandLine.Xplat` directly can happen in both .NET Framework and .NET Core but is not a real scenario.

The ideal situation would be detecting that the .NET CLI is calling us, then prepend `dotnet `. However, this is probably not worth the effort.